packages/@aws-cdk/aws-codebuild/lib/source.ts

```diff
@@ -768,6 +815,14 @@ export interface BitBucketSourceProps extends ThirdPartyGitSourceProps {
    * @example 'aws-cdk'
    */
   readonly repo: string;
+
+  /**
+   * This parameter is used for the `name` parameter in the Bitbucket commit status.
+   *
+   * @example "My build #$CODEBUILD_BUILD_NUMBER"
+   * @default "AWS CodeBuild $AWS_REGION ($PROJECT_NAME)"
+   */
```

Comment:
Let's also add links to the CodeBuild docs about this functionality:

```suggestion
   *
   * @see https://docs.aws.amazon.com/codebuild/latest/userguide/create-project-cli.html#cli.source.buildstatusconfig.context
   * @see https://docs.aws.amazon.com/codebuild/latest/userguide/build-env-ref-env-vars.html
   */
```